lib/roo_on_rails/checks/sidekiq/sidekiq.rb

```diff
@@ -0,0 +1,45 @@
+require 'roo_on_rails/checks/base'
+require 'thor'
+
+module RooOnRails
+  module Checks
+    module Sidekiq
+      class Sidekiq < Base
```

Comment:
Seems a bit weird having the same class name as the containing module (both `Sidekiq`). Maybe it should be called `Procfile` as that seems to be what it's doing?